.travis.yml

```diff
@@ -1,18 +1,12 @@
 language: python
-env:
```

Comment:
To make your life easier right now?

Reply:
yes, tests were passing thrice and we need to go a little bit faster 😅